evalml/automl/automl_search.py

```diff
@@ -244,8 +244,7 @@ def __init__(self,
             'search_order': [],
             'errors': []
         }
-        self.random_state = get_random_state(random_state)
-        self.random_seed = get_random_seed(self.random_state)
+        self.random_state = get_random_seed(random_state)
```

Comment:
I think it could be more clear if this variable was renamed to `self.random_seed` and all references to `random_state` be changed to `random_seed` but isn't a big issue and could go in a further cleanup issue.